package main

func main() {
	panic("这是自定义异常")
}
